### PR TITLE
pantheon.switchboard-plug-about: Backport AppStream 1.0 build fix

### PR DESCRIPTION
**Pull request by bobby285271**
## Description of changes

It's done conditionally so we are fine.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/pantheon/apps/switchboard-plugs/about/default.nix [hunk 1]
```diff
@@ -29,6 +30,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-MJybc2yAchU6qMqkoRz45QdhR7bj/UFk2nyxcBivsHI=";
   };
 
+  patches = [
+    # Add support for AppStream 1.0
+    # https://github.com/elementary/switchboard-plug-about/pull/275
+    (fetchpatch {
+      url = "https://github.com/elementary/switchboard-plug-about/commit/72d7da13da2824812908276751fd3024db2dd0f8.patch";
+      sha256 = "sha256-R7oW3mL77/JNqxuMiqxtdMlHWMJgGRQBBzVeRiqx8PY=";
```
Comment (SuperSandro2000) [inline pkgs/desktops/pantheon/apps/switchboard-plugs/about/default.nix:38]:
```suggestion
      hash = "sha256-R7oW3mL77/JNqxuMiqxtdMlHWMJgGRQBBzVeRiqx8PY=";
```

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by K900** [comment]
Should we maybe get all the appstream 1.0 changes into one PR? I can separate mine out.

**Comment by bobby285271** [comment]
Maybe, we also have stuff that needs [new appstream](https://gitlab.gnome.org/GNOME/gnome-software/-/merge_requests/1810/diffs#diff-content-0cc1139e3347f573ae1feee5b73dbc8a8a21fcfa) in GNOME 46 (beta in February) :shrug:

**Comment by bobby285271** [comment]
I guess I will land this one for now since the branch name really is `upd/pantheon`.

---

I ran a very initial nixpkgs-review on b08decd3a6a8c8d59f0e954080efb397ab710d41 (this pr + commits from kde6 pr + small fixup for libadwaita):

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>lua52Packages.nfd</li>
    <li>lua53Packages.nfd</li>
    <li>lua54Packages.nfd</li>
    <li>textpieces</li>
  </ul>
</details>
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>appimage-run-tests</li>
  </ul>
</details>
<details>
  <summary>128 packages failed to build:</summary>
  <ul>
    <li>bottles</li>
    <li>bottles-unwrapped</li>
    <li>cemu</li>
    <li>citations</li>
    <li>flexoptix-app</li>
    <li>gamemode</li>
    <li>gamemode.dev</li>
    <li>gamemode.lib</li>
    <li>gamemode.man</li>
    <li>gamemode.static</li>
    <li>gnome.gnome-software</li>
    <li>heroic</li>
    <li>kodiPackages.libretro-2048</li>
    <li>kodiPackages.libretro-fuse</li>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-nestopia</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>libretro.atari800</li>
    <li>libretro.beetle-gba</li>
    <li>libretro.beetle-lynx</li>
    <li>libretro.beetle-ngp</li>
    <li>libretro.beetle-pce</li>
    <li>libretro.beetle-pce-fast</li>
    <li>libretro.beetle-pcfx</li>
    <li>libretro.beetle-psx</li>
    <li>libretro.beetle-psx-hw</li>
    <li>libretro.beetle-saturn</li>
    <li>libretro.beetle-supafaust</li>
    <li>libretro.beetle-supergrafx</li>
    <li>libretro.beetle-vb</li>
    <li>libretro.beetle-wswan</li>
    <li>libretro.blastem</li>
    <li>libretro.bluemsx</li>
    <li>libretro.bsnes</li>
    <li>libretro.bsnes-hd</li>
    <li>libretro.bsnes-mercury</li>
    <li>libretro.bsnes-mercury-balanced</li>
    <li>libretro.bsnes-mercury-performance</li>
    <li>libretro.citra</li>
    <li>libretro.desmume</li>
    <li>libretro.desmume2015</li>
    <li>libretro.dolphin</li>
    <li>libretro.dosbox</li>
    <li>libretro.dosbox-pure</li>
    <li>libretro.eightyone</li>
    <li>libretro.fbalpha2012</li>
    <li>libretro.fbneo</li>
    <li>libretro.fceumm</li>
    <li>libretro.flycast</li>
    <li>libretro.fmsx</li>
    <li>libretro.freeintv</li>
    <li>libretro.fuse</li>
    <li>libretro.gambatte</li>
    <li>libretro.genesis-plus-gx</li>
    <li>libretro.gpsp</li>
    <li>libretro.gw</li>
    <li>libretro.handy</li>
    <li>libretro.hatari</li>
    <li>libretro.mame</li>
    <li>libretro.mame2000</li>
    <li>libretro.mame2003</li>
    <li>libretro.mame2003-plus</li>
    <li>libretro.mame2010</li>
    <li>libretro.mame2015</li>
    <li>libretro.mame2016</li>
    <li>libretro.melonds</li>
    <li>libretro.mesen</li>
    <li>libretro.mesen-s</li>
    <li>libretro.meteor</li>
    <li>libretro.mgba</li>
    <li>libretro.mupen64plus</li>
    <li>libretro.neocd</li>
    <li>libretro.nestopia</li>
    <li>libretro.np2kai</li>
    <li>libretro.nxengine</li>
    <li>libretro.o2em</li>
    <li>libretro.opera</li>
    <li>libretro.parallel-n64</li>
    <li>libretro.pcsx-rearmed</li>
    <li>libretro.pcsx2</li>
    <li>libretro.picodrive</li>
    <li>libretro.play</li>
    <li>libretro.ppsspp</li>
    <li>libretro.prboom</li>
    <li>libretro.prosystem</li>
    <li>libretro.puae</li>
    <li>libretro.quicknes</li>
    <li>libretro.same_cdi</li>
    <li>libretro.sameboy</li>
    <li>libretro.scummvm</li>
    <li>libretro.smsplus-gx</li>
    <li>libretro.snes9x</li>
    <li>libretro.snes9x2002</li>
    <li>libretro.snes9x2005</li>
    <li>libretro.snes9x2005-plus</li>
    <li>libretro.snes9x2010</li>
    <li>libretro.stella</li>
    <li>libretro.stella2014</li>
    <li>libretro.swanstation</li>
    <li>libretro.tgbdual</li>
    <li>libretro.thepowdertoy</li>
    <li>libretro.tic80</li>
    <li>libretro.twenty-fortyeight</li>
    <li>libretro.vba-m</li>
    <li>libretro.vba-next</li>
    <li>libretro.vecx</li>
    <li>libretro.virtualjaguar</li>
    <li>libretro.yabause</li>
    <li>libsForQt5.discover</li>
    <li>mangohud</li>
    <li>mangohud.doc</li>
    <li>mangohud.man</li>
    <li>pantheon.appcenter</li>
    <li>pantheon.elementary-feedback</li>
    <li>pantheon.elementary-onboarding</li>
    <li>prismlauncher</li>
    <li>prismlauncher-qt5</li>
    <li>prismlauncher-qt5-unwrapped</li>
    <li>prismlauncher-unwrapped</li>
    <li>pureref</li>
    <li>radicle-upstream</li>
    <li>retroarch</li>
    <li>retroarchBare</li>
    <li>retroarchFull</li>
    <li>setzer</li>
    <li>surge</li>
    <li>timeular</li>
  </ul>
</details>
<details>
  <summary>545 packages built:</summary>
  <ul>
    <li>Sylk</li>
    <li>adapta-gtk-theme</li>
    <li>ajour</li>
    <li>alfis</li>
    <li>altair</li>
    <li>amberol</li>
    <li>anytype</li>
    <li>appimage-run</li>
    <li>appstream</li>
    <li>appstream.dev</li>
    <li>appstream.installedTests</li>
    <li>ashpd-demo</li>
    <li>authenticator</li>
    <li>autokey</li>
    <li>autokey.dist</li>
    <li>bada-bib</li>
    <li>badlion-client</li>
    <li>banking</li>
    <li>baobab</li>
    <li>bazecor</li>
    <li>beekeeper-studio</li>
    <li>beeper</li>
    <li>bespokesynth</li>
    <li>bespokesynth-with-vst2</li>
    <li>betterdiscord-installer</li>
    <li>blackbox-terminal</li>
    <li>blanket</li>
    <li>bloomrpc</li>
    <li>blueberry</li>
    <li>blueprint-compiler</li>
    <li>boatswain</li>
    <li>bookworm</li>
    <li>bootstrap-studio</li>
    <li>breath-theme</li>
    <li>budgie.budgie-control-center</li>
    <li>budgie.budgie-control-center.debug</li>
    <li>budgie.budgie-desktop</li>
    <li>budgie.budgie-desktop-with-plugins</li>
    <li>budgie.budgie-gsettings-overrides</li>
    <li>budgie.magpie</li>
    <li>budgie.magpie.debug</li>
    <li>budgie.magpie.dev</li>
    <li>budgie.magpie.devdoc</li>
    <li>budgiePlugins.budgie-analogue-clock-applet</li>
    <li>budgiePlugins.budgie-user-indicator-redux</li>
    <li>buttercup-desktop</li>
    <li>cambalache</li>
    <li>caprine-bin</li>
    <li>cartridges</li>
    <li>catnip-gtk4</li>
    <li>cavalier</li>
    <li>celeste</li>
    <li>celluloid</li>
    <li>chrysalis</li>
    <li>cider</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.cinnamon-session</li>
    <li>cinnamon.muffin</li>
    <li>cinnamon.muffin.dev</li>
    <li>cinnamon.muffin.man</li>
    <li>cinnamon.nemo</li>
    <li>cinnamon.nemo-fileroller</li>
    <li>cinnamon.nemo-python</li>
    <li>cinnamon.nemo-with-extensions</li>
    <li>cinnamon.nemo.dev</li>
    <li>cinnamon.pix</li>
    <li>cinnamon.warpinator</li>
    <li>cinnamon.xapp</li>
    <li>cinnamon.xapp.dev</li>
    <li>cinnamon.xreader</li>
    <li>cinnamon.xviewer</li>
    <li>clapper</li>
    <li>codux</li>
    <li>collision</li>
    <li>colloid-kde</li>
    <li>confy</li>
    <li>conjure</li>
    <li>contrast</li>
    <li>cozy-drive</li>
    <li>crypto-org-wallet</li>
    <li>d-spy</li>
    <li>d-spy.dev</li>
    <li>d-spy.lib</li>
    <li>davinci-resolve</li>
    <li>davinci-resolve-studio</li>
    <li>deepin.dde-launchpad</li>
    <li>deja-dup</li>
    <li>denaro</li>
    <li>denemo</li>
    <li>devdocs-desktop</li>
    <li>dialect</li>
    <li>diebahn</li>
    <li>dino</li>
    <li>dippi</li>
    <li>done</li>
    <li>dropbox-cli</li>
    <li>dropbox-cli.nautilusExtension</li>
    <li>dynamic-wallpaper</li>
    <li>eartag</li>
    <li>easyeffects</li>
    <li>easyeffects.debug</li>
    <li>eiciel</li>
    <li>eiciel.nautilusExtension</li>
    <li>elastic</li>
    <li>electron-mail</li>
    <li>electronplayer</li>
    <li>emblem</li>
    <li>emulsion-palette</li>
    <li>endeavour</li>
    <li>epiphany</li>
    <li>evince</li>
    <li>evince.dev</li>
    <li>evince.devdoc</li>
    <li>eyedropper</li>
    <li>fclones-gui</li>
    <li>firefly-desktop</li>
    <li>flare-signal</li>
    <li>flatpak</li>
    <li>flatpak-builder</li>
    <li>flatpak-builder.doc</li>
    <li>flatpak-builder.installedTests</li>
    <li>flatpak-builder.man</li>
    <li>flatpak.dev</li>
    <li>flatpak.devdoc</li>
    <li>flatpak.doc</li>
    <li>flatpak.installedTests</li>
    <li>flatpak.man</li>
    <li>flowtime</li>
    <li>fluent-reader</li>
    <li>fluffychat</li>
    <li>fm</li>
    <li>foliate</li>
    <li>fractal</li>
    <li>fragments</li>
    <li>framesh</li>
    <li>fspy</li>
    <li>furtherance</li>
    <li>g4music</li>
    <li>geopard</li>
    <li>giara</li>
    <li>glom</li>
    <li>glom.dev</li>
    <li>glom.devdoc</li>
    <li>glom.doc</li>
    <li>glom.lib</li>
    <li>gnome-browser-connector</li>
    <li>gnome-builder</li>
    <li>gnome-builder.devdoc</li>
    <li>gnome-console</li>
    <li>gnome-decoder</li>
    <li>gnome-extension-manager</li>
    <li>gnome-feeds</li>
    <li>gnome-firmware</li>
    <li>gnome-frog</li>
    <li>gnome-obfuscate</li>
    <li>gnome-podcasts</li>
    <li>gnome-secrets</li>
    <li>gnome-solanum</li>
    <li>gnome-tecla</li>
    <li>gnome-text-editor</li>
    <li>gnome-tour</li>
    <li>gnome-usage</li>
    <li>gnome.file-roller</li>
    <li>gnome.ghex</li>
    <li>gnome.ghex.dev</li>
    <li>gnome.ghex.devdoc</li>
    <li>gnome.gnome-bluetooth</li>
    <li>gnome.gnome-bluetooth.dev</li>
    <li>gnome.gnome-bluetooth.devdoc</li>
    <li>gnome.gnome-bluetooth.man</li>
    <li>gnome.gnome-calculator</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-characters</li>
    <li>gnome.gnome-chess</li>
    <li>gnome.gnome-clocks</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-control-center.debug</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-font-viewer</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-logs</li>
    <li>gnome.gnome-mahjongg</li>
    <li>gnome.gnome-maps</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-panel-with-modules</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-session.debug</li>
    <li>gnome.gnome-session.sessions</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-shell.debug</li>
    <li>gnome.gnome-shell.devdoc</li>
    <li>gnome.gnome-sound-recorder</li>
    <li>gnome.gnome-sudoku</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gnome-weather</li>
    <li>gnome.gpaste</li>
    <li>gnome.metacity</li>
    <li>gnome.mutter</li>
    <li>gnome.mutter.debug</li>
    <li>gnome.mutter.dev</li>
    <li>gnome.mutter.devdoc</li>
    <li>gnome.mutter.man</li>
    <li>gnome.mutter43</li>
    <li>gnome.mutter43.debug</li>
    <li>gnome.mutter43.dev</li>
    <li>gnome.mutter43.devdoc</li>
    <li>gnome.mutter43.man</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>gnome.nautilus-python.dev</li>
    <li>gnome.nautilus-python.devdoc</li>
    <li>gnome.nautilus-python.doc</li>
    <li>gnome.nautilus.dev</li>
    <li>gnome.nautilus.devdoc</li>
    <li>gnome.nixos-gsettings-overrides</li>
    <li>gnome.polari</li>
    <li>gnome.sushi</li>
    <li>gnome.zenity</li>
    <li>gnomeExtensions.easyeffects-preset-selector</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.gsconnect.installedTests</li>
    <li>gnomeExtensions.gtk4-desktop-icons-ng-ding</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>gnonograms</li>
    <li>gpu-viewer</li>
    <li>gradience</li>
    <li>grapejuice</li>
    <li>grapejuice.dist</li>
    <li>graphite-kde-theme</li>
    <li>gtkcord4</li>
    <li>gtranslator</li>
    <li>halftone</li>
    <li>hamsket</li>
    <li>headlines</li>
    <li>health</li>
    <li>helio-workstation</li>
    <li>helvum</li>
    <li>hifile</li>
    <li>hqplayer-desktop</li>
    <li>hqplayer-desktop.doc</li>
    <li>hypnotix</li>
    <li>icon-library</li>
    <li>identity</li>
    <li>imaginer</li>
    <li>immersed-vr</li>
    <li>impression</li>
    <li>iotas</li>
    <li>irccloud</li>
    <li>jbrowse</li>
    <li>jetbrains-toolbox</li>
    <li>joplin-desktop</li>
    <li>junction</li>
    <li>karlender</li>
    <li>keet</li>
    <li>khronos</li>
    <li>kodelife</li>
    <li>kodi-cli</li>
    <li>kodiPackages.steam-launcher</li>
    <li>komikku</li>
    <li>kooha</li>
    <li>lact</li>
    <li>latte-dock</li>
    <li>layan-kde</li>
    <li>lbry</li>
    <li>ldtk</li>
    <li>ledger-live-desktop</li>
    <li>lens</li>
    <li>libadwaita</li>
    <li>libadwaita.dev</li>
    <li>libadwaita.devdoc</li>
    <li>libpanel</li>
    <li>libpanel.dev</li>
    <li>libpanel.devdoc</li>
    <li>libsForQt5.applet-window-appmenu</li>
    <li>libsForQt5.appstream-qt</li>
    <li>libsForQt5.appstream-qt.dev</li>
    <li>libsForQt5.appstream-qt.installedTests</li>
    <li>libsForQt5.bismuth</li>
    <li>libsForQt5.flatpak-kcm</li>
    <li>libsForQt5.kde-cli-tools</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.kdev-php</li>
    <li>libsForQt5.kdev-python</li>
    <li>libsForQt5.kdevelop</li>
    <li>libsForQt5.kdevelop-unwrapped</li>
    <li>libsForQt5.khotkeys</li>
    <li>libsForQt5.khotkeys.bin</li>
    <li>libsForQt5.khotkeys.dev</li>
    <li>libsForQt5.kinfocenter</li>
    <li>libsForQt5.kmenuedit</li>
    <li>libsForQt5.krohnkite</li>
    <li>libsForQt5.kwin-dynamic-workspaces</li>
    <li>libsForQt5.kwin-tiling</li>
    <li>libsForQt5.kzones</li>
    <li>libsForQt5.packagekit-qt</li>
    <li>libsForQt5.parachute</li>
    <li>libsForQt5.plasma-bigscreen</li>
    <li>libsForQt5.plasma-browser-integration</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5.plasma-disks</li>
    <li>libsForQt5.plasma-disks.bin</li>
    <li>libsForQt5.plasma-disks.dev</li>
    <li>libsForQt5.plasma-mobile</li>
    <li>libsForQt5.plasma-remotecontrollers</li>
    <li>libsForQt5.plasma-workspace</li>
    <li>libsForQt5.plasma-workspace.dev</li>
    <li>libsForQt5.powerdevil</li>
    <li>libsForQt5.systemsettings</li>
    <li>libsForQt5.systemsettings.bin</li>
    <li>libsForQt5.systemsettings.dev</li>
    <li>libsForQt5.xdg-desktop-portal-kde</li>
    <li>lightdm-slick-greeter</li>
    <li>lightly-boehs</li>
    <li>lightly-qt</li>
    <li>livecaptions</li>
    <li>losslesscut-bin</li>
    <li>loupe</li>
    <li>lua51Packages.nfd</li>
    <li>luajitPackages.nfd</li>
    <li>ludusavi</li>
    <li>lunar-client</li>
    <li>lunatask</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lxi-tools-gui</li>
    <li>malcontent-ui</li>
    <li>malcontent-ui.dev</li>
    <li>malcontent-ui.lib</li>
    <li>marktext</li>
    <li>mate.marco</li>
    <li>mate.mate-applets</li>
    <li>mate.mate-control-center</li>
    <li>mate.mate-indicator-applet</li>
    <li>mate.mate-media</li>
    <li>mate.mate-netbook</li>
    <li>mate.mate-notification-daemon</li>
    <li>mate.mate-panel</li>
    <li>mate.mate-power-manager</li>
    <li>mate.mate-screensaver</li>
    <li>mate.mate-sensors-applet</li>
    <li>mate.mate-settings-daemon-wrapped</li>
    <li>mate.mate-tweak</li>
    <li>mate.mate-tweak.dist</li>
    <li>mate.mate-utils</li>
    <li>mathpix-snipping-tool</li>
    <li>megaglest</li>
    <li>mendeley</li>
    <li>mepo</li>
    <li>metadata-cleaner</li>
    <li>meteo</li>
    <li>mission-center</li>
    <li>mobilecoin-wallet</li>
    <li>mockoon</li>
    <li>mojave-gtk-theme</li>
    <li>molotov</li>
    <li>monitor</li>
    <li>monophony</li>
    <li>motrix</li>
    <li>mousai</li>
    <li>museeks</li>
    <li>muzika</li>
    <li>mycrypto</li>
    <li>nautilus-open-any-terminal</li>
    <li>nautilus-open-any-terminal.dist</li>
    <li>neo4j-desktop</li>
    <li>netease-cloud-music-gtk</li>
    <li>newsflash</li>
    <li>nordic</li>
    <li>nordic.sddm</li>
    <li>notable</li>
    <li>notejot</li>
    <li>notesnook</li>
    <li>notion-app-enhanced</li>
    <li>nuclear</li>
    <li>openlens</li>
    <li>openraPackages_2019.engines.bleed</li>
    <li>openraPackages_2019.engines.playtest</li>
    <li>openraPackages_2019.engines.release</li>
    <li>openraPackages_2019.mods.ca</li>
    <li>openraPackages_2019.mods.d2</li>
    <li>openraPackages_2019.mods.dr</li>
    <li>openraPackages_2019.mods.gen</li>
    <li>openraPackages_2019.mods.kknd</li>
    <li>openraPackages_2019.mods.mw</li>
    <li>openraPackages_2019.mods.ra2</li>
    <li>openraPackages_2019.mods.raclassic</li>
    <li>openraPackages_2019.mods.rv</li>
    <li>openraPackages_2019.mods.sp</li>
    <li>openraPackages_2019.mods.ss</li>
    <li>openraPackages_2019.mods.ura</li>
    <li>openraPackages_2019.mods.yr</li>
    <li>osu-lazer-bin</li>
    <li>oversteer</li>
    <li>p3x-onenote</li>
    <li>paleta</li>
    <li>pantheon.elementary-code</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-photos</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.epiphany</li>
    <li>pantheon.file-roller-contract</li>
    <li>pantheon.gala</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-about</li>
    <li>pantheon.switchboard-plug-applications</li>
    <li>pantheon.switchboard-plug-bluetooth</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-a11y</li>
    <li>pantheon.wingpanel-indicator-bluetooth</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-indicator-network</li>
    <li>pantheon.wingpanel-indicator-nightlight</li>
    <li>pantheon.wingpanel-indicator-notifications</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-indicator-session</li>
    <li>pantheon.wingpanel-indicator-sound</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>passes</li>
    <li>pdfmm</li>
    <li>pentobi</li>
    <li>phoc</li>
    <li>phosh</li>
    <li>phosh-mobile-settings</li>
    <li>pika-backup</li>
    <li>planify</li>
    <li>plata-theme</li>
    <li>plexamp</li>
    <li>plots</li>
    <li>plots.dist</li>
    <li>pods</li>
    <li>polypane</li>
    <li>portfolio-filemanager</li>
    <li>protontricks</li>
    <li>protontricks.dist</li>
    <li>protonup-qt</li>
    <li>pwvucontrol</li>
    <li>python310Packages.xapp</li>
    <li>python311Packages.xapp</li>
    <li>qogir-kde</li>
    <li>qt6Packages.appstream-qt</li>
    <li>qt6Packages.appstream-qt.dev</li>
    <li>qt6Packages.appstream-qt.installedTests</li>
    <li>qt6Packages.packagekit-qt</li>
    <li>raider</li>
    <li>rambox</li>
    <li>raven-reader</li>
    <li>remnote</li>
    <li>requestly</li>
    <li>resources</li>
    <li>rnote</li>
    <li>rtfm</li>
    <li>rymcast</li>
    <li>saleae-logic-2</li>
    <li>satty</li>
    <li>sbclPackages.cl-gtk4_dot_adw</li>
    <li>schemes</li>
    <li>session-desktop</li>
    <li>sfizz</li>
    <li>shortwave</li>
    <li>showmethekey</li>
    <li>simplex-chat-desktop</li>
    <li>snapshot</li>
    <li>snippetpixie</li>
    <li>sonixd</li>
    <li>spacedrive</li>
    <li>spot</li>
    <li>spotify</li>
    <li>spotifywm</li>
    <li>ssb-patchwork</li>
    <li>station</li>
    <li>steam</li>
    <li>steam-rom-manager</li>
    <li>steam-small</li>
    <li>steamPackages.steam-fhsenv-without-steam</li>
    <li>sticky</li>
    <li>symbolic-preview</li>
    <li>sysprof</li>
    <li>sysprof.dev</li>
    <li>sysprof.lib</li>
    <li>tagger</li>
    <li>tailor-gui</li>
    <li>tangram</li>
    <li>tev</li>
    <li>text-engine</li>
    <li>tilix</li>
    <li>timeshift</li>
    <li>timeshift-minimal</li>
    <li>timeshift-unwrapped</li>
    <li>todoist-electron</li>
    <li>touchosc</li>
    <li>trayscale</li>
    <li>trezor-suite</li>
    <li>trigger-control</li>
    <li>tuba</li>
    <li>tusk</li>
    <li>upscayl</li>
    <li>utterly-nord-plasma</li>
    <li>vala-language-server</li>
    <li>valent</li>
    <li>vcv-rack</li>
    <li>via</li>
    <li>vial</li>
    <li>video-trimmer</li>
    <li>vimPlugins.vCoolor-vim</li>
    <li>vimix-gtk-themes</li>
    <li>wacomtablet</li>
    <li>warp</li>
    <li>watchmate</li>
    <li>webfontkitgenerator</li>
    <li>whatip</li>
    <li>whitesur-gtk-theme</li>
    <li>whitesur-kde</li>
    <li>wike</li>
    <li>winetricks</li>
    <li>wingpanel-indicator-ayatana</li>
    <li>wl-color-picker</li>
    <li>wootility</li>
    <li>wordbook</li>
    <li>xdg-desktop-portal</li>
    <li>xdg-desktop-portal-gnome</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xdg-desktop-portal-xapp</li>
    <li>xdg-desktop-portal.installedTests</li>
    <li>xed-editor</li>
    <li>xlights</li>
    <li>xmonad_log_applet_mate</li>
    <li>xplayer</li>
    <li>ytmdesktop</li>
    <li>zecwallet-lite</li>
    <li>zettlr</li>
    <li>zrythm</li>
    <li>zulip</li>
  </ul>
</details>

The many `libretro.*` failures are caused by `gamemode`, which `appstreamcli validate` fails. I imagine most other breakages are failing with the same reason. It might take me a while (?) to make fixes, I try to.

**Comment by bobby285271** [comment]
I took a look at the `appstreamcli validate` thing, actually I will prefer fetchpatch https://github.com/ximion/appstream/commit/0e12d9840bef383c55b439d350649d95d8ef88f1. It makes no sense to make `developer-name-tag-deprecated` fail the build when `developer` is an unknown tag in appstream 0.15.x which is still shipped in e.g. ubuntu 22.04.

**Comment by bobby285271** [comment]
- https://github.com/NixOS/nixpkgs/pull/273297

(I started another nixpkgs-review. Also tried to unbreak darwin builds)

**Pull request merged**